src/scripts/hooks/utils/locatorCommands.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     return lib.uiControlExecuteLocator(ui5Selector, null, timeout, rootElement);
   },
 
-  getUI5Property: async function (propName, selectorOrElement) {
+  getUI5Property: async function (propName, element) {
```

Comment:
why did we change to element only here? Is it just bc we already got the element at this level?

Reply:
will fix that, that code shouldn't have been changed